naloge/dectree3.tex

```diff
@@ -11,6 +11,40 @@
 \podnaslov{Odločitveno drevo}
 \end{slika}
 \end{vprasanje}
+
 \begin{odgovor}
+Izračunajmo najprej pričakovane vrednosti v vozliščih $C$, $F$ in $G$
+odločitvenega drevesa s slike~\fig.
+\begin{alignat*}{2}
+C &= 0.5 \cdot (40 p + 20) &&= 20p + 10 \\
+F &= 50 \cdot p + 10 \cdot (1-p) &&= 60 p - 10 \\
+G &= 30 \cdot p - 30 \cdot (1-p) &&=  -60p + 30
```

Comment:
Tale dva izračuna nista pravilna, tako da bo potrebno popraviti tudi nadaljnje izračune in odločitve.

Reply:
Vidim sedaj, da so rezultati pravi, narobe so pa napisane enačbe, iz katerih naj bi jih dobili.